core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -436,12 +436,23 @@ oppia.directive('conversationSkin', [
             }
 
             if (ExplorationPlayerStateService.isStateTerminal(stateName)) {
-              ExplorationRecommendationsService.getRecommendedSummaryDicts(
-                ExplorationPlayerStateService.getAuthorRecommendedExpIds(
-                  stateName),
-                function(summaries) {
-                  $scope.recommendedExplorationSummaries = summaries;
-                });
+              var lastParameter = UrlService.getUrlParams();
```

Comment:
Write

    var parentExplorationId = UrlService.getParentExplorationId()

and write a comprehensive Karma test for the new method (which should return null if there's no parent arg in the URL).

Reply:
Changed variable, will add karma test.